Assignment3/Marksheet.java

```diff
@@ -0,0 +1,115 @@
+import java.util.*;
+
+/*
+Marksheet class is to input the grades (0 to 100) of n students
+and calculate the average of all grades, calculate the percentage
+of students passed, find the minimum and maximum of all grades.
+
+@author Saurabh Aswal
+*/
+class Marksheet
+{ 
+	static int n;					//Total number of Students.
+	double grades[];				//Array of grades of the students.
+	
+	/*
+	 * To input the grades and keep them in an array 'grades'.
+	 * 
+	 * @param x is the number of students of type int.
+	 */
+	void fillArray(int x)
```

Comment:
Lack of uniformity in the code, global variable is passed as parameter to method.